packages/suite/src/reducers/suite/suiteReducer.ts

```diff
@@ -21,6 +21,7 @@ interface Flags {
     // pinCompleted: boolean;
     // passphraseCompleted: boolean;
     discreetModeCompleted: boolean;
+    forcedRememberDevice: boolean;
```

Comment:
it would be better to have as optional string `forcedRememberDevice?: string` where value would be `device.state`

then you don't have to add this field everywhere (initialState, tests etc)